tests/test_contract_dependencies/packages/default_author/contracts/stub_0/__init__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
```

Comment:
These packages should be located in `tests/data` as `stub_0` and `stub_1` dirs. Or under the `packages` structure there.